temporalio/activity.py

```diff
@@ -260,6 +287,11 @@ def info() -> Info:
     return _Context.current().info()
 
 
+def cancellation_details() -> Optional[ActivityCancellationDetails]:
+    """Cancellation details of the currenct activity, if any"""
```

Comment:
We may want to say when this is first set with a value, it is never unset and the value is never changed

Reply:
added to the docstring